pypdf/generic/_utils.py

```diff
@@ -7,6 +7,11 @@
 from ._base import ByteStringObject, TextStringObject
 
 
+# This function converts a hex code to an RGB tuple. It takes a string as input,
+# and returns a tuple of floats. The string is expected to be of the form
+# "#RRGGBB", where RR, GG, and BB are two-digit hexadecimal numbers. The
+# returned tuple contains the red, green, and blue values, each normalized to
+# the range [0, 1].
```

Comment:
This could be a docstring and it could be shorter. It also looks as if ChatGPT generated it.